### ISSUE TITLE
Threshold() and slope_at_proportion_correct() broken

### ISSUE THREAD

**Issue by guillermoaguilar** [wichmann-lab/python-psignifit#137]
This concerns the method `threshold()` from the Results object.

When attempting to read a threshold value different than the one set in the fit, it errors:

```
result.threshold(0.8)
Traceback (most recent call last):

  Cell In[14], line 1
    result.threshold(0.8)

  File ~/git/python-psignifit/psignifit/_result.py:99 in threshold
    new_threshold = sigmoid.inverse(proportion_correct, self.parameter_estimate['threshold'],

  File ~/git/python-psignifit/psignifit/sigmoids.py:128 in inverse
    raise ValueError(f'prop_correct={prop_correct} has to be between {gamma} and {1 - lambd}.')

ValueError: prop_correct=0.8 has to be between 3.3448045998592124e-07 and 0.5.
```

This does not make sense as the default argument of `threshold()` is `unscaled=False`, which should mean that I'm reading the function in the range from gamma to 1-lambda.. in this case from 0.5 to 1.0 (I'm using the basic example data).

**Comment by guillermoaguilar** [comment]
The wished behavior should be as in [the matlab function getThreshold()](https://github.com/wichmann-lab/psignifit/wiki/How-to-Get-Thresholds-and-Slopes#obtaining-threshold-values), default is unscaled as it is the one that makes sense to the experimenter.

**Comment by guillermoaguilar** [comment]
Also broken is the `slope_at_proportion_correct` method

```
result.slope_at_proportion_correct(0.8)
Traceback (most recent call last):

  Cell In[9], line 1
    result.slope_at_proportion_correct(0.8)

  File ~/git/python-psignifit/psignifit/_result.py:141 in slope_at_proportion_correct
    stimulus_levels = self.threshold(proportion_correct, unscaled, return_ci=False)

  File ~/git/python-psignifit/psignifit/_result.py:99 in threshold
    new_threshold = sigmoid.inverse(proportion_correct, self.parameter_estimate['threshold'],

  File ~/git/python-psignifit/psignifit/sigmoids.py:128 in inverse
    raise ValueError(f'prop_correct={prop_correct} has to be between {gamma} and {1 - lambd}.')

ValueError: prop_correct=0.8 has to be between 3.3448045998592124e-07 and 0.5.
```

FYI method `slope` is OK.

**Comment by otizonaizit** [comment]
I wouldn't say this is _broken_. These are manually generated exceptions. What happens if you remove those checks in the code? They have been added by @dekuenstle , as far as `git blame` is concerned, and he may remember why he added these checks. But as far as I can tell, the code treats these parameters as variables and they are all settable in the configuration.
@guillermoaguilar : if after removing the checks everything still works and the numbers returned are correct, that I'd say it's done, right?

**Comment by guillermoaguilar** [comment]
OK I digged into it, it was an error in the function arguments, gamma and lambda were in reverse order. I fixed it in PR #139

**Comment by pberkes** [comment]
Fixed
